### DISCUSSION TITLE
Safe to update to npm 8.1.1 ?

### DISCUSSION THREAD

**Discussion by francisp2** [Koenkk/zigbee2mqtt#9344] [General]
I saw this compiling Zigbee2MQTT : 

```
npm notice New major version of npm available! 7.24.1 -> 8.1.1
npm notice Changelog: https://github.com/npm/cli/releases/tag/v8.1.1
npm notice Run npm install -g npm@8.1.1 to update!
```
is it safe, or I stay better on 7.24.1 ?

**Comment by lux73** [comment]
interesting Question. Two Days ago v8.1.2 is released: https://www.npmjs.com/package/npm?activeTab=versions

anybody knows it is safe to update?

**Comment by lux73** [comment]
i have updated **npm** to **v8.1.2** few minutes ago - no Problems so far

**Comment by francisp2** [comment]
Thanks.
